database/engine/rrdengine.h

```diff
@@ -99,6 +100,27 @@ struct generic_io_descriptor {
     struct completion *completion;
 };
 
+struct extent_cache_element {
+    struct extent_info *extent; /* The ABA problem is avoided with the help of fileno below */
+    unsigned fileno;
+    struct extent_cache_element *prev; /* LRU */
+    struct extent_cache_element *next; /* LRU */
```

Comment:
/* MRU */?

Reply:
Hmm, maybe I should just say `LRU list`?